src/System.Net.Security/tests/FunctionalTests/SslStreamStreamToStreamTest.cs

```diff
@@ -14,27 +16,102 @@ namespace System.Net.Security.Tests
 {
     public class SslStreamStreamToStreamTest
     {
+        private readonly byte[] sampleMsg = Encoding.UTF8.GetBytes("Sample Test Message");
+        private const int TaskWaitTime = 15; // seconds
+
+        #region Tests
+
         [Fact]
         public void SslStream_StreamToStream_Authentication_Success()
         {
+            SslStream clientSslStream = null;
```

Comment:
Please make this code consistent with other tests. The helper functions, while enabling code reuse make this code harder to read and so I would remove them.